Versionize.Tests/WorkingCopyTests.cs

```diff
@@ -93,6 +93,46 @@ public void ShouldExitIfWorkingCopyIsDirty()
         _testPlatformAbstractions.Messages[0].ShouldBe($"Repository {_testSetup.WorkingDirectory} is dirty. Please commit your changes:\nNewInIndex: hello.txt");
     }
 
+    [Fact]
+    public void ShouldExitIfChangelogPathsPointingToSameLocation()
```

Comment:
Will you move this test to a new file called `ConfigProviderTests` in the empty Config folder?

Reply:
Also thought on this, because of this request also for review.